layout-management-webapps/src/main/resources/locale/portlet/layoutManagement/SiteNavigationPortlet_en.properties

```
@@ -56,4 +56,11 @@ siteNavigation.label.invalidLink=Invalid link
 siteNavigation.label.enterUrl=Enter URL
 siteNavigation.label.newTab=New tab
 siteNavigation.label.sameTab=Same tab
-siteNavigation.label.pageTemplate=Select page template
\ No newline at end of file
+siteNavigation.label.pageTemplate=Select page template
+siteNavigation.label.selectExistingPage=Select an existing page from
+siteNavigation.label.navigationsSuggester.searchPlaceholder=Enter a site name
+siteNavigation.label.navigationsSuggester.noDataLabel=No data
+siteNavigation.label.pageSuggester.searchPlaceholder=Enter a page name
+siteNavigation.label.pageSuggester.noDataLabel=No data
+siteNavigation.label.allSites=All sites
+siteNavigation.label.sitesOrSpace=Select a site or a space
```

```suggestion
siteNavigation.label.selectSites=Select sites
```
it should be general IMO "select sites" not specific for a site or a space, to check with @mzorai